Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/ManipulationHandler.cs

```diff
@@ -279,9 +321,30 @@ private void Start()
             {
                 hostTransform = transform;
             }
+
+            transformHelper = this.EnsureComponent<TransformHelper>();
+            transformHelper.Initialize(hostTransform);
+            transformHelper.SetScaleLimits(scaleMinimum, scaleMaximum);
         }
         #endregion MonoBehaviour Functions
 
+        #region Public Methods
+
+        /// <summary>
+        /// Sets the minimum/maximum scale for the manipulation handler at runtime.
+        /// </summary>
+        /// <param name="min">Minimum scale</param>
+        /// <param name="max">Maximum scale</param>
+        /// <param name="relativeToInitialState">If true the values will be multiplied by scale of target at startup. If false they will be in absolute local scale.</param>
+        public void SetScaleLimits(float min, float max, bool relativeToInitialState = true)
+        {
+            scaleMaximum = max;
+            scaleMinimum = min;
```

Comment:
it feels a bit wrong to me that we set the scale min and max here but not retain the info about this being relative to the initial state. so if you would call start() again after setscalelimits has been called with relativeToInitialState is true , the scale would suddenly change because it's interpreting those values as relativeToInitialState the second time this is called.